index.html

```diff
@@ -10,6 +10,43 @@
 
     
   <h1>Pat's Famous Salmon Cookies</h1>
+  <p>We know what you're thinking,salmon and cookies shouldn't be in the same sentence. <br>
+  How could you take two things that are so good on their own but together might smell a little something like cat litter.<br>
+ Well we dared to go where no one would ever and actually created something incredibly delicious. <br>
+ Pat, our infamous leader got the idea from a dare one late night cooking with friends. <br>
+ A few hours later everyone in the room gladly accepted losing that dare. <br>
+ Here at Pat's we want to bring worlds together that may not ever meet and create great memories through cookies. <br>
+  We promise when you tell your cousin in Florida that you had the best salmon cookie you ever had it won't be a lie!</p>
+
+  
   <img src="images/salmon.png" alt="salmon">
+  <div id"storediv"=>
```

Comment:
This id has the equals sign in the wrong place--it isn't affecting the functionality of your site, just FYI